Related-projects.rst

```diff
@@ -0,0 +1,43 @@
+
+Related projects
+================
+
+There are a number of other free software projects to provide a *Linux experience* for mobile devices. Some of these projects are actively working on moving to Halium. All of them are kindred spirits in the quest for Linux on mobile. Not the least, they have valuable resources available at their respective sites about porting to mobile devices.
+
+
+LuneOS
+------
+
+`LuneOS <https://webos-ports.org>`_ is based on Open webOS. It is provided by the WebOS Ports team.
+The system has it's legacy in projects by LG, HP and Palm. There are ongoing conversations about a cooperation with Halium.
+
+
+Plasma mobile
+-------------
+
+`Plasma-mobile  <https://plasma-mobile.org>`_ is based on the Plasma shell, developed by the KDE community. Plasma mobile is available on Halium.
+
+
+postmarketOS
+------------
+
+`PostmarketOS <https://postmarketos.org>`_ is based on Alpine Linux. It has a selection of multiple phone interfaces. A main difference to Halium is that postmarketOS targets upstream kernels. This brings the advantage of more up to date software. It has the disadvantage that device drivers which are only available as binaries for old kernels can not be used.
+
+
+SailfishOS
+----------
+
+`Sailfish OS <https://sailfishos.org/>`_ is based on the Mer Project. The user interface is developed by Jolla and not open source. The technical architecture should be largely compatible with Halium.
+
+
+UBports
+-------
+
+`UBports <https://ubports.com>`_ is based on Unity 8. It is the continuation of the earlier Ubuntu Touch project from Canonical. UBports is actively working on making their future releases available on Halium.
```

Comment:
*tiny* nitpick: The software is called Ubuntu Touch, the entity which controls it is UBports. It would be appropriate to use Ubuntu Touch in this case.